src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleModel.scala

```diff
@@ -1,13 +1,19 @@
 package org.broadinstitute.dsde.workbench.sam.google
 
-import org.broadinstitute.dsde.workbench.model.ErrorReport
+import cats.effect.IO
+import org.broadinstitute.dsde.workbench.model.{ErrorReport, ErrorReportSource}
 import spray.json.DefaultJsonProtocol
 
 object SamGoogleModelJsonSupport {
   import DefaultJsonProtocol._
   import org.broadinstitute.dsde.workbench.model.ErrorReportJsonSupport._
 
-  implicit val SyncReportItemFormat = jsonFormat3(SyncReportItem)
+  implicit val SyncReportItemFormat = jsonFormat3(SyncReportItem.apply)
 }
 
-case class SyncReportItem(operation: String, email: String, errorReport: Option[ErrorReport])
+final case class SyncReportItem(operation: String, email: String, errorReport: Option[ErrorReport])
+
+object SyncReportItem {
```

Comment:
I don't love the name `SyncReportItem`, or maybe it needs a comment or something to explain what it is.  Perhaps out of scope for this ticket, but if our goal is to improve the synching stuff, maybe it isn't?  Up to you.